src/architecture/utilities/orbitalMotion.c

```diff
@@ -107,7 +103,7 @@ void  hill2rv(double *rc_N, double *vc_N, double *rho_H, double *rhoPrime_H, dou
  *   rho_H: deputy Hill position vector
  *   rhoPrime_H: deputy Hill velocity vector
  */
```

Comment:
I believe you intended to squash this whole commit

Reply:
Thanks. Forgot to reword the commit message.